.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2.1
+orbs:
+  node: circleci/node@3.0.1
+
+jobs:
+  build-lint-test:
+    parameters:
+      version:
+        type: string
+    docker:
+      - image: circleci/node:<< parameters.version >>
+    steps:
+      - checkout
+      - run: npm install
```

Comment:
I suggest running `npm ci` here instead. This way, if something is off in the `package-lock.json`, this will catch it.

Reply:
Addressed in #278